courses/utils.py

```diff
@@ -0,0 +1,81 @@
+"""
+Utility functions for the courses app
+"""
+import re
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+
+NUMBER_SEASON_MAP = {
+    1: 'Spring',
+    2: 'Summer',
+    3: 'Fall'
+}
+
+
+def parse_edx_key(edx_course_key):
+    """
+    Args:
+        edx_course_key (str): An edX course key (CourseRun.edx_course_key)
+
+    Returns:
```

Comment:
Can you put the type right after `Returns:` instead of within parentheses in the description? It helps my IDE parse these if it matches the schema exactly